src/lib/init.js

```diff
@@ -112,15 +137,9 @@ export function init(configUpdates) {
 			// Request lists
 			return Promise.all([
 				store,
-				fetchGlobalVendorList().then(store.updateVendorList),
-				fetchPurposeList().then(store.updateCustomPurposeList)
+				fetchGlobalVendorList().then(store.updateVendorList)
 			]).then((params) => {
-				//Update consent data if cookie on global domain exists
-				if (store.persistedGlobalVendorConsentData) {
-					store.mergeVendorConsentsFromGlobalCookie();
-					store.mergePurposeConsentsFromGlobalCookie();
-				}
-
+				cmpApi.customCommands = createCommands(store);
```

Comment:
To trzeba zmienić,  tak aby custom komendy były dostępne przed załadowaniem listy vendorów, zaraz po inicjalizacji CMP API.
Listę vendorów należy pobrać wewnątrz komendy za pomocą API. Obecnie zanim pobierze się lista vendorów nie będzie obsługiwana żadna customowa komenda.

Reply:
done